### DISCUSSION TITLE
Excuse me, I'd like to consult if werkzeug 0.15.6 version is affected by CVE-2023-46136

### DISCUSSION THREAD

**Discussion by kongyi182** [pallets/werkzeug#2849] [Q&A]
Excuse me, I'd like to consult if werkzeug 0.15.6 version is affected by CVE-2023-46136

**Comment by pgjones** [comment]
Werkzeug 0.15.6 is not supported, and is really quite old now. The code the CVE refers too wasn't present then, but this issue could still be present.
